### DISCUSSION TITLE
Corner Pin Fx

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6470] [Ideas]
### Is your feature request related to a problem? Please describe

I know that free distortion Fx already exists, but I find it too complicated for a task that should be simple. Having 8 distortion points overlapping each other makes it quite confusing in my opinion. It's like trying to unwind a cable in a mountain of cables 🤣 

### Describe the solution you'd like

A simple Corner Pin Fx with four points in each corner that can be dragged to distort the image could be fine.

**Comment by flurick** [comment]
I'm curious how often the source rectangle is actually animated.
If its mostly for perspective, perhaps the animation tools Shear mode could be tweaked to be more usable?
